package.json

```diff
@@ -5,16 +5,19 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "lint": "eslint .",
-    "start": "node index.js",
+    "lint": "tsc --noEmit && eslint --ext .js,.ts .",
+    "start": "node dist/index.js",
     "migrate": "node-pg-migrate --database-url-var=RCTF_DATABASE_URL",
-    "build": "sh client/build.sh",
-    "dev:client": "preact watch --src client/src --template client/index.html",
-    "dev": "nodemon index.js",
-    "test": "nyc --skip-full ava",
+    "build:client": "sh client/build.sh",
+    "build:ts": "tsc && yarn copy-static",
+    "build": "yarn build:ts && yarn build:client",
+    "watch:client": "preact watch --src client/src --template client/index.html",
+    "watch:server": "nodemon dist/index.js",
+    "watch:ts": "tsc -w",
+    "dev": "yarn build:ts && concurrently -k -t \"HH:mm:ss.SSS\" -p \"[{time}]\" -c \"cyan,green,red\" \"yarn watch:client\" \"yarn watch:server\" \"yarn watch:ts\"",
```

Comment:
maybe 1) change to a `watch:*` and 2) have output labeled by which watch job in addition to time? (2 might not be necessary since they're already different colors)

Reply:
`dev` is more declarative cause it's the thing you're gonna be using for development.

Reply:
no I meant replace the watch: in the concurrently call